ContributionGuidelines.md

```diff
@@ -0,0 +1,140 @@
+# [DOC2PEN](https://doc2pen.smaranjitghose.codes/)📝
+#### Doc2pen is an Open Source Software that converts your Typed Docs into Handwritten Docs.
+
+# Contribution Guidelines [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/esta/issues) 🤝🏽🍀:
+
+## File Structure of the Project 🗃️📂
+
+### [Public Directory Structure](https://github.com/smaranjitghose/doc2pen/tree/master/public) 🧮✨
```

Comment:
Public Directory of not of any interest. We are just concerned with src

Reply:
So Shall I remove public directory?